include/libdnf5/rpm/package_query.hpp

```diff
@@ -281,6 +281,18 @@ class PackageQuery : public PackageSet {
     /// @since 5.0
     //
     // @replaces libdnf/sack/query.hpp:method:addFilter(int keyname, int cmp_type, const char *match) - cmp_type = HY_PKG_PROVIDES
+    void filter_provides(const std::string & pattern, libdnf5::sack::QueryCmp cmp_type = libdnf5::sack::QueryCmp::EQ) {
```

Comment:
If we go with this version the docs need an update, it says `A vector of strings ..` but it takes just a string.